### PR TITLE
feature/Task02-homeWork

### PR DESCRIPTION
**Pull request by daniel250592**

### CODE REVIEW

**Comment by daniel250592** [comment]
HomeWork

File: backend/src/main/java/pl/sdacademy/simple/model/CarAttribute.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@Entity
+@Table(name = "car_attribute")
+public class CarAttribute {
+
+    @Id
+    private long id;
+
+    private String attribute;
+
+   @ManyToMany(mappedBy = "atributes")
```
Comment (softak-code) [inline backend/src/main/java/pl/sdacademy/simple/model/CarAttribute.java:19]: Not necessary Bi-direction relation

File: backend/src/main/java/pl/sdacademy/simple/model/CarAttribute.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private Set<CarEntity> cars = new HashSet<>();
+
+    public Set<CarEntity> getCars() {
+        return cars;
+    }
+
+    public CarAttribute(long id, String attribute) {
+        this.id = id;
+        this.attribute = attribute;
+    }
```
Comment (softak-code) [inline backend/src/main/java/pl/sdacademy/simple/model/CarAttribute.java:29]: No getters, why?

File: backend/src/main/java/pl/sdacademy/simple/model/CarEntity.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    public CarEntity(long id, String producer, String model, String color, BigDecimal price, boolean automatic) {
+        this.id = id;
+        this.producer = producer;
+        this.model = model;
+        this.color = color;
+        this.price = price;
+        this.automatic = automatic;
+    }
```
Comment (softak-code) [inline backend/src/main/java/pl/sdacademy/simple/model/CarEntity.java:43]: Also, not getters, why?

File: backend/src/main/java/pl/sdacademy/simple/repositories/CarAttributeRepository.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+import pl.sdacademy.simple.model.CarAttribute;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityTransaction;
+
+public class CarAttributeRepository {
+    private final EntityManager entityManager;
+
+    public CarAttributeRepository(EntityManager entityManager) {
```
Comment (softak-code) [inline backend/src/main/java/pl/sdacademy/simple/repositories/CarAttributeRepository.java:11]: This class is not necessary regarding our requirements.

File: backend/src/main/java/pl/sdacademy/simple/repositories/CarEntityRepository.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import javax.persistence.EntityTransaction;
+
+public class CarEntityRepository {
+    private final EntityManager entityManager;
+
+    public CarEntityRepository(EntityManager entityManager) {
+        this.entityManager = entityManager;
+    }
+
+    public boolean save(CarEntity carEntity) {
```
Comment (softak-code) [inline backend/src/main/java/pl/sdacademy/simple/repositories/CarEntityRepository.java:15]: We won't create any cars, so this method is not necessary.

File: backend/src/main/java/pl/sdacademy/simple/repositories/Demo.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
+package pl.sdacademy.simple.repositories;
```
Comment (softak-code) [inline backend/src/main/java/pl/sdacademy/simple/repositories/Demo.java:1]: Move to test area.

**Pull request closed without merging**